### PR TITLE
Adds Donations "create" Action for Stripe Checkout

### PR DESCRIPTION
**Pull request by ribordy**
This replaces #13. The only thing missing here is the config to pass the new environment variables I added through to the deployed environment. If someone could point me towards whether they belong in `.github/workflows/*.yml`, `serverless.yml`, or both, that'd be great!

### CODE REVIEW

**Comment by mojowen** [comment]
🙌

They should be in both the `serverless.yml` and the two deploy files.

I've added two secrets you can reference `STRIPE_KEY` for `deploy.yml` and `STAGING_STRIPE_KEY` for `test.yml`

**Comment by ribordy** [comment]
@mojowen I think we'll need one for `DONATION_PRODUCT_ID` too--for `CLIENT_APP_URL` I think hard-coding is fine; for `test` would that be `next.polls.pizza`?

File: src/controller/DonationsController.ts [hunk 1]
```diff
@@ -20,28 +22,64 @@ export class DonationsController {
       return `Webhook Error: ${err.message}`;
     }
 
-    const { type, data } = request.body;
+    // const { type, data } = request.body;
```
Comment (mojowen) [inline src/controller/DonationsController.ts:25]: Can just delete this right?

**Review by mojowen — APPROVED** [review]

File: src/controller/DonationsController.ts (hunk 1 above)
Comment (ribordy) [reply src/controller/DonationsController.ts:25]: Yep--I think that came in from a merge, but it was unused--I'll remove!

**Pull request merged**